modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/test/java/com/liferay/dynamic/data/mapping/form/builder/internal/converter/DDMFormRuleToDDMFormRuleModelConverterTest.java

```diff
@@ -379,32 +381,58 @@ private List<String> _extractCallFunctionParameters(String callFunction) {
 	}
 
 	private void _setUpDDMExpressionFactory() throws Exception {
-		Field field = ReflectionUtil.getDeclaredField(
-			_ddmExpressionFactoryImpl.getClass(),
-			"ddmExpressionFunctionTracker");
+		DDMExpressionFunctionTracker ddmExpressionFunctionTracker =
+			Mockito.mock(DDMExpressionFunctionTracker.class);
 
-		field.set(
-			_ddmExpressionFactoryImpl, new DDMExpressionFunctionTrackerImpl());
+		Mockito.when(
+			ddmExpressionFunctionTracker.getDDMExpressionFunctionFactories(
+				Matchers.any())
+		).thenReturn(
+			HashMapBuilder.<String, DDMExpressionFunctionFactory>put(
```

Comment:
Do you think we can buy lines here? Perhaps create a method that iterates a collection of keys and puts the same value into the map? If you think this isn't worth it, I trust your judgement and we can move forward with this. 😄